server/src/resolvers/queries/index.ts

```diff
@@ -42,6 +42,23 @@ async function nftByTokenId(token_id: string, ctx: Context) {
   };
 }
 
+async function nftByOwnerId(owner_address: string, ctx: Context) {
+  const { nftData, ownerData } = await extractNftData(ctx);
+  const owners = ownerData.filter((kv: any) => kv.value === owner_address);
+  if (owners.length === 0) return [];
+  const tokens = nftData.filter((kv: any) =>
+    owners.find((owner: any) => kv.value.token_id === owner.key)
```

Comment:
I believe it should be kv.key === owner.key

Reply:
Ledger key is token_ids